src/tools/CMakeLists.txt

```diff
@@ -20,6 +22,8 @@ install(TARGETS ceph_scratchtoolpp DESTINATION bin)
 
 add_executable(ceph_radosacl radosacl.cc)
 target_link_libraries(ceph_radosacl librados global)
+set_target_properties(ceph_radosacl PROPERTIES
```

Comment:
you're right `ceph_radosacl`, `ceph_scratchtool`, and `ceph_test_libcephfs` don't need to set the INSTALL_RPATH property. I wrongly identified them.

> why you don't set the `INSTALL_RPATH` for all executables?
I don't understand what you mean, currently `INSTALL_RPATH` is already set for all binaries, what I'm currently doing is setting an empty 'INSTALL_RPATH` for all binaries that don't depend on any internal (also being built) target